Yubico.YubiKey/tests/unit/Yubico/YubiKey/YubikeyDeviceInfoTests.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright 2023 Yubico AB
```

Comment:
2024

Reply:
Ah, will get it in the next merge. This header should get set automatically on a solution code format/cleanup in future commits